11-parallelism/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+//! 11 Parallelism
+//! --------------
+//!
+//! Welcome to eleven step of this Rust workshop.
+//!
+//! This step focuses on basic multi threading API in Rust
+//!
+//! ## Thread
+//!
+//! To create a new thread, we call the `thread::spawn` function and pass it a **closure**.
+//!
+//! NOTE:
+//! Calling join on the handle blocks the thread currently running
+//! until the thread represented by the handle terminates.
+//!
+//! ```rust
+//! use std::thread;
+//!
+//! let other_thread = thread::spawn(|| {
+//!     return 1 + 1
```

Comment:
`return` ?

Reply:
I think people will forgot how return without  `;` work. So I prefer let them focus on the `thread` part and the `closure` and put a return to make it more clear for them.

Reply:
Then, I prefer hint comment

Reply:
Done